features/functests/performance/performance.go

```diff
@@ -1,73 +1,214 @@
 package performance
 
 import (
+	"fmt"
 	"os/exec"
 	"strings"
+	"time"
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
+
+	ocv1 "github.com/openshift/api/config/v1"
+	k8sv1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/fields"
 	"k8s.io/apimachinery/pkg/labels"
 
 	"github.com/openshift-kni/baremetal-deploy/features/functests/utils/clients"
-	"github.com/openshift-kni/baremetal-deploy/features/functests/utils/namespace"
-
-	k8sv1 "k8s.io/api/core/v1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-const testNamespace = "perfomancetest"
+const (
+	perfTestNamespace                       = "perfomancetest"
+	perfWorkerNodesLabel                    = "node-role.kubernetes.io/worker-rt="
+	perfMachineConfigDaemonContainer        = "machine-config-daemon"
+	perfClusterNodeTuningOperatorNamespaces = "openshift-cluster-node-tuning-operator"
+	perfSysctlTimeout                       = 240
+	perfSysctlPollInterval                  = 2
+)
 
 var mcKernelArguments = []string{"isolcpus"}
 
 var _ = Describe("performance", func() {
-	beforeAll(func() {
-		namespace.Create(testNamespace, clients.K8s)
-		namespace.Clean(testNamespace, clients.K8s)
+	// 00-tuned-network-latency.yaml related verification
+	var _ = Context("Network latency parameters adjusted by the Node Tuning Operator", func() {
+		It("Should contain configuration injected through the 00-tuned-network-latency.yaml template", func() {
+			sysctlMap := map[string]string{
+				"net.core.busy_read":              "50",
+				"net.core.busy_poll":              "50",
+				"net.ipv4.tcp_fastopen":           "3",
+				"kernel.numa_balancing":           "0",
+				"kernel.sched_min_granularity_ns": "10000000",
+				"vm.dirty_ratio":                  "10",
+				"vm.dirty_background_ratio":       "3",
+				"vm.swappiness":                   "10",
+				"kernel.sched_migration_cost_ns":  "5000000",
+			}
+			const perfNodeNetworkLatencyTuned = "openshift-node-network-latency"
+			_, err := clients.CntoConfig.TunedV1().Tuneds(perfClusterNodeTuningOperatorNamespaces).Get(perfNodeNetworkLatencyTuned,metav1.GetOptions{})
```

Comment:
im not a big fan of adding a lot of dependencies for this one liner (if this is the case) if the check turns out to be un-needed then lets remove it.

Reply:
@yanirq , perhaps. The other way  - execute the oc command and parse the output. Lets the CI to complete all the tests successfully and afterwards decide about more convenient solution.